_maps/map_files/AsteroidStation/AsteroidStation.dmm

```diff
@@ -2322,7 +2322,6 @@
 "aSH" = (/obj/structure/cable/yellow{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/structure/cable/yellow{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/floor/plating/airless{icon_state = "warnplatecorner"; dir = 1},/area/engine/engineering)
 "aSI" = (/turf/simulated/wall,/area/engine/engineering)
 "aSJ" = (/obj/structure/closet/emcloset,/turf/simulated/floor/plating{icon_state = "warnplatecorner"; dir = 4},/area/engine/engineering)
-"aSK" = (/obj/machinery/suit_storage_unit/engine,/turf/simulated/floor/plasteel{dir = 8; icon_state = "yellow"},/area/engine/engineering)
```

Comment:
I don't understand how this could have happened, I literally updated to master, prepared map merge, went in and manually deleted the christmas stuff via the dang drop down menu, then map merged.

It's possible that this specific tile loadout is already defined to a different letter combination than "aSK" and this was just cleaning up the doubled bit.